package.json

```
@@ -2,6 +2,15 @@
   "name": "broadcast-channel",
   "version": "4.0.0",
   "description": "A BroadcastChannel that works in New Browsers, Old Browsers, WebWorkers and NodeJs",
+  "exports": {
+    "node": {
+      "import": "./dist/es/index.js",
+      "default": "./dist/es5node/index.js"
+    },
+    "browser": {
+      "default": "./dist/lib/index.es5.js"
+    }
```

You should also export `package.json` as some tools will try to read it and need access to it (e.g. `rollup-plugin-svelte`)
```suggestion
    },
    "./package.json": "./package.json"
```